templates/magento2/vhost-server-includes.conf.j2

```diff
@@ -0,0 +1,172 @@
+#jinja2: lstrip_blocks: True
+# {{ ansible_managed }}
+
+set $MAGE_ROOT {{ release_pub_dir }};
+set $MAGE_MODE {{ mage_run_mode }};
+
+if ($http_x_forwarded_proto = "https") {
+    set $is_https "1";
+}
+
+# NOTE: if the request is over http
+if ($http_x_forwarded_proto != 'https') {
+    set $unsecure true;
+}
+
+# NOTE: if the request is from aws alb
+if ($http_x_amzn_trace_id) {
+    set $unsecure "${unsecure}+aws";
```

Comment:
@lvampa would you mind initializing this before using it within an `if` block?

Reply:
Ugggh, this is what I get for refactoring

Reply:
The nginx error log thanks you very much.